### DISCUSSION TITLE
browsing worktree..

### DISCUSSION THREAD

**Discussion by cypeak** [jesseduffield/lazygit#4036] [Q&A]
i know this may sound stupid, but how can i just browse the current worktree dirs/files and see their status (i.e. authors/last edit date, history, etc..)

**Comment by mark2185** [comment]
You can't, unfortunately, it's not something `lazygit` offers.

It doesn't sound stupid, it's a valid question :)

**Comment by joshribakoff** [comment]
Very interested in this! I'm confused though, if its not in there what am I seeing? 
<img width="901" height="375" alt="Screenshot 2025-12-14 at 12 26 57 PM" src="https://github.com/user-attachments/assets/2e06caa9-29da-46fc-b6ac-e8cad6fcbd26" />

**Reply by stefanhaller** [reply]
The question was about browsing the files in your worktree; not about browsing your worktrees.

**Comment by sanrishi** [comment]
> i know this may sound stupid, but how can i just browse the current worktree dirs/files and see their status (i.e. authors/last edit date, history, etc..)

No it is a valid and important question !
